experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -20,12 +20,24 @@
 CACHE_HOST="http://bazel-cache.default:8080"
 
 make_bazel_rc () {
+    # this is the default for recent releases but we set it explicitly
+    # since this is the only hash our cache supports
     echo "startup --host_jvm_args=-Dbazel.DigestFunction=sha256" >> $1
+    # caching args for build
+    # - use remote strategy for all the things
     echo "build --spawn_strategy=remote" >> $1
     echo "build --strategy=Javac=remote" >> $1
     echo "build --genrule_strategy=remote" >> $1
-    echo "build --remote_rest_cache=${CACHE_HOST}" >> $1
+    # - use the http cache ...
+    echo "build --remote_http_cache=${CACHE_HOST}" >> $1
```

Comment:
`--remote_rest_cache` was deprecated in `0.10.0`, this is the correct flag name now.